packages/plugins/connections/connection-mongodb/src/connections/MongoDBCollection/MongoDBAggregation/schema.js

```diff
@@ -0,0 +1,26 @@
+export default {
```

Comment:
Hi @correiarmjoao 
We need to add the license comment block to all the schema files. It's only the json files which do not include a license block.